### PR TITLE
Disabling signature provider caching

### PR DESCRIPTION
**Pull request by JeffAshton**

### CODE REVIEW

File: src/D2L.Security.OAuth2/Keys/Default/D2LSecurityToken.cs [hunk 1]
```diff
@@ -46,7 +46,9 @@ internal sealed partial class D2LSecurityToken : SecurityKey {
 				this,
 				GetSignatureAlgorithm()
 			) {
-				CryptoProviderFactory = new D2LCryptoProviderFactory()
+				CryptoProviderFactory = new D2LCryptoProviderFactory {
+					CacheSignatureProviders = false
```
Comment (j3parker) [inline src/D2L.Security.OAuth2/Keys/Default/D2LSecurityToken.cs:50]: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/wiki/Caching-in-Microsoft.IdentityModel
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1605

**Review by j3parker — APPROVED** [review]

**Comment by j3parker** [comment]
@omsmith  ^ ?

**Review by omsmith — APPROVED** [review]
I would prefer this gets set inside `D2LCryptoProviderFactory`, but otherwise looks good.

**Comment by omsmith** [comment]
This doesn't prevent the cache itself from getting made, so it's still going to leak tasks.

**Comment by JeffAshton** [comment]
> This doesn't prevent the cache itself from getting made, so it's still going to leak tasks.

well that sucks then

**Pull request closed without merging**